wpilibNewCommands/src/main/native/include/frc2/command/CommandPtr.h

```diff
@@ -189,9 +189,20 @@ class CommandPtr final {
    * @param parallel the commands to run in parallel
    * @return the decorated command
    */
-  [[nodiscard]]
+  [[nodiscard]] [[deprecated("Deprecated use DeadlineFor instead")]]
```

Comment:
The word "deprecated" in a deprecation message is redundant.
These messages are idiomatically phrased as "Replace with DeadlineFor".